docs/api/version3.md

```diff
@@ -541,25 +537,36 @@ Request payload:
 
 ```jsonc
 {
-  "data": {
-    "transactionFeeEstimates": {
-      "minFee": "135000",
-      "accountInitializationFee": {
-        "tokenID": "0000000000000000",
-        "amount": "5000000"
-      },
-      "messageFee": { // Optional: Available only for a 'token:transferCrossChain' transaction
-        "tokenID": "0000000000000000",
-        "amount": "343000"
-      }
+    "data": {
+        "transaction": {
+            "fee": {
+                "tokenID": "0000000000000000",
+                "minimum": "172000", // byteFee + all additionalFees
+                "priority": { // optional - available only when priority values differ
+                    "low": "135000",
+                    "medium": "270000",
+                    "high": "405000"
+                } 
+            },
+            "params": {
+                "messageFee": { // Optional: Available only for a 'token:transferCrossChain' transaction
+                    "tokenID": "0000000000000000",
+                    "amount": "343000"
+                }
+            }
+        }
     },
-    "dynamicFeeEstimates": {
-      "low": "135000",
-      "medium": "270000",
-      "high": "405000"
+    "meta": {
```

Comment:
We might have to update the `meta` structure slightly to accommodate the ´params.messageFee`breakdown into `ccmByteFee` and `accountInitializationFee` for `token:transferCrossChain` transactions. Let's connect offline later to discuss this.